libvast/src/column_index.cpp

```diff
@@ -95,20 +95,20 @@ caf::error column_index::init() {
   if (exists(filename_)) {
     detail::value_index_inspect_helper tmp{index_type_, idx_};
     if (auto err = load(sys_, filename_, last_flush_, tmp)) {
-      VAST_ERROR("unable to load value index from disk", sys_.render(err));
+      VAST_ERROR(this, "unable to load value index from disk", sys_.render(err));
```

Comment:
Seems like the sentence doesn't quite read properly: `this, "failed to load..."`?